syntax = "proto3";

package server;

message SrvHandshake {
  int32 id = 1;
}